pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -126,6 +136,40 @@ func (c TargetConfigController) sync() error {
 	return nil
 }
 
+func isRequiredConfigPresent(config []byte) error {
```

Comment:
cluster-kube-apiserver-operator also has tests for this. Better copy them as well.

Reply:
done